### PR TITLE
Bunty chakraborty

### PR DESCRIPTION
**Pull request by buntychakraborty**

### CODE REVIEW

File: OASys/src/main/java/com/oasys/main/model/Question.java [hunk 1]
```diff
@@ -75,4 +75,13 @@ public void setOption4(String option4) {
 		this.option4 = option4;
 	}
 
+	@Override
+	public String toString() {
+		return "Question [questionId=" + questionId + ", questionText=" + questionText + ", option1=" + option1
```
Comment (Subham-Mallick) [inline OASys/src/main/java/com/oasys/main/model/Question.java:80]: @buntychakraborty
Yaha question a json change hoga. wo ek baar thik karle

File: OASys/src/main/java/com/oasys/main/model/Score.java [hunk 2]
```diff
@@ -0,0 +1,70 @@
...
+import java.util.Map;
+import com.fasterxml.jackson.annotation.JsonAnyGetter;
+import com.fasterxml.jackson.annotation.JsonAnySetter;
+import com.fasterxml.jackson.annotation.JsonIgnore;
+import com.fasterxml.jackson.annotation.JsonInclude;
+import com.fasterxml.jackson.annotation.JsonProperty;
+import com.fasterxml.jackson.annotation.JsonPropertyOrder;
+
+@JsonInclude(JsonInclude.Include.NON_NULL)
+@JsonPropertyOrder({
```
Comment (Subham-Mallick) [inline OASys/src/main/java/com/oasys/main/model/Score.java:14]: @buntychakraborty 
iske attributes change honge

File: OASys/src/main/java/com/oasys/main/model/UserResponseDTO.java [hunk 3]
```diff
@@ -0,0 +1,29 @@
+
```
Comment (Subham-Mallick) [inline OASys/src/main/java/com/oasys/main/model/UserResponseDTO.java:1]: @buntychakraborty 
is file ka reason ?

**Review by Subham-Mallick — CHANGES_REQUESTED** [review]
Question json final ho jae then tu necessary changes karke wapis bhejna. And saab kuch ke lie @mitush ko bhi sync me rakhna.

File: OASys/src/main/java/com/oasys/main/model/UserResponseDTO.java (hunk 3 above)
Comment (buntychakraborty) [reply OASys/src/main/java/com/oasys/main/model/UserResponseDTO.java:1]: User ta response bhej raha hai

**Pull request closed without merging**